assets/templates/login.html

```diff
@@ -101,15 +101,17 @@ <h5>INGRESA A DOPPLER</h5>
           <small>©2018 Doppler LLC. Todos los derechos reservados.</small>
         </footer>
       </article>
-      <section class="feature-panel bg--login">
-        <article class="feature-content">
-          <h6>Formularios de suscripción</h6>
-          <h1>Suma contactos a tus Listas con Formularios personalizados</h1>
-          <p>Clásicos y pop-ups con suscripción Simple o Doble Opt-In. ¡Tú eliges cómo quieres que luzcan, qué datos solicitar y dónde ubicarlos!</p>
-        </article><!-- article -->
-        <figure class="content-img">
-          <img src="img/login-es.png" alt="Formularios">
-        </figure>
+      <section class="feature-panel">
+        <div class="feature-panel--bg">
+          <article class="feature-content">
+            <h6>Formularios de suscripción</h6>
+            <h1>Suma contactos a tus Listas con Formularios personalizados</h1>
+            <p>Clásicos y pop-ups con suscripción Simple o Doble Opt-In. ¡Tú eliges cómo quieres que luzcan, qué datos solicitar y dónde ubicarlos!</p>
+          </article><!-- article -->
+          <figure class="content-img">
+            <img src="img/login-es.png" alt="Formularios">
```

Comment:
If I am not wrong, the background is also customizable, did you take that into account?

Reply:
what background? i use 2 images. Column background and content image

Reply:
Claro, hay dos imágenes, el fondo (por ahora amarillo y violeta) y la imagen (la notebook).

Pero solo veo una en el HTML, si la otra está en los estilos no se va a poder modificar, bah, se podría con estilos inline.

See me escapa algo?

Reply:
en la webapp esta inline! aca en este pr no esta, pero verdad podria agregarlo para quede mas claro

Reply:
listo actualizado